pom.xml

```diff
@@ -467,6 +467,7 @@
             <exclude>**/*.dbf</exclude>
             <exclude>**/*.cnf</exclude>
             <exclude>**/*.access_log</exclude>
+            <exclude>**/success.ftl</exclude>
```

Comment:
Please fix the license header instead of excluding the file.

Reply:
Done.  I renamed the file `success.html`.